FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_IPv6_Demo/IPv6_Multi_WinSim_demo/TCPEchoClient_SingleTasks.c

```diff
@@ -70,6 +70,9 @@
 /* The number of instances of the echo client task to create. */
     #define echoNUM_ECHO_CLIENTS          ( 1 )
 
+/* To enable the use of zero copy interface of the TCP sockets */
+    #define USE_TCP_ZERO_COPY             ( 0 )
```

Comment:
Should we add this flag to FreeRTISIPConfig.h?

Reply:
Keeping this flag to just the demo file will help not to build the entire source files when just the zero copy macro is changed. Another reason is to make it similar to the convention we follow in existing demos:[ UDPEchoClient_SingleTasks.c: USE_ZERO_COPY](https://github.com/FreeRTOS/FreeRTOS/blob/0b5cd0866590b63346b95464a45186b74d35f916/FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_IPv6_Demo/IPv6_Multi_WinSim_demo/UDPEchoClient_SingleTasks.c#L53)